### ISSUE TITLE
Fixing Navbar

### ISSUE THREAD

**Issue by Ardent10** [backdrop-ops/docs.backdropcms.org#190]
The current Navbar right box is showing both the Welcome and the GitHub and other links are  showing in column manner which is kind of odd right now. 
![image](https://user-images.githubusercontent.com/73132031/157712971-04357a0b-443c-4a8d-a20f-dc82a7b8094c.png)

Instead of showing them in column we can show them in a  single row which will be much better.

**Comment by stpaultim** [comment]
Thanks for the feedback. @Ardent10 

I agree that the header is a bit awkward and could use some help. My first thought was that there is already too much in a single row and that maybe "Documentation and API source code for Backdrop CMS" should be moved to a second line below the site title.

Currently, this header isn't working good on midsized screens. 

Here is how other Backdrop CMS headers look.

https://backdropcms.org/
![image](https://user-images.githubusercontent.com/3144571/157785770-091c4c9d-c8f5-4fe6-bd27-5900c586480a.png)

https://forum.backdropcms.org/
![image](https://user-images.githubusercontent.com/3144571/157785834-94c99dc3-97f8-4f74-bb45-964709a22c3c.png)

https://events.backdropcms.org/
![image](https://user-images.githubusercontent.com/3144571/157785896-3d239d22-307e-4c46-a448-48b2d12a0523.png)

**Comment by Ardent10** [comment]
> Thanks for the feedback. @Ardent10
> 
> I agree that the header is a bit awkward and could use some help. My first thought was that there is already too much in a single row and that maybe "Documentation and API source code for Backdrop CMS" should be moved to a second line below the site title.
> 
> Currently, this header isn't working good on midsized screens.
> 
> Here is how other Backdrop CMS headers look.
> 
> https://backdropcms.org/ ![image](https://user-images.githubusercontent.com/3144571/157785770-091c4c9d-c8f5-4fe6-bd27-5900c586480a.png)
> 
> https://forum.backdropcms.org/ ![image](https://user-images.githubusercontent.com/3144571/157785834-94c99dc3-97f8-4f74-bb45-964709a22c3c.png)
> 
> https://events.backdropcms.org/ ![image](https://user-images.githubusercontent.com/3144571/157785896-3d239d22-307e-4c46-a448-48b2d12a0523.png)

1) Yes, We can also do it this way. We can move the "Documentation and API source code for Backdrop CMS" in the second line just below the Title "Backdrop CMS Documentation". 
2) As per the Responsiveness is concerned I will make a new Issue after this issue is resolved. 
3) I can also see few more things that can be improved in the website. I will create a new issue for them. 
I should be working on this issue now. Will that be ok @stpaultim ?

**Comment by stpaultim** [comment]
@Ardent10 - Working on design issues such as this is a bit more complicated, because as an opensource community we sometimes need to get more opinions before we make changes. Let's get the opinion of @jenlampton as to whether or not this is a good thing to work on. 

She will likely reply soon. This might not be a good issue for someone new to our issue queue, but I would like to hear from @jenlampton before we decide.

**Comment by Ardent10** [comment]
> @Ardent10 - Working on design issues such as this is a bit more complicated, because as an opensource community we sometimes need to get more opinions before we make changes. Let's get the opinion of @jenlampton as to whether or not this is a good thing to work on.
> 
> She will likely reply soon. This might not be a good issue for someone new to our issue queue, but I would like to hear from @jenlampton before we decide.

Sure @stpaultim

**Comment by jenlampton** [comment]
Yeah, there's nothing here to really work on yet as we'll need a design to be approved before we can change anything. The small elements at the top right of the page were balanced against the larger logo, so in order to separate them we probably need to adjust something else to compensate (like the size?).  I can bring this issue to our designer for some suggestions. (Note: I am not a designer, I could be wrong, but that's why I like to ask one:)

**Comment by jenlampton** [comment]
I was looking at the different headers on our sites again today. The "backdrop" in our logo should not be capitalized, so Ive taken a stab at cleaning up the headers of all the sites to be (more) consistent. What would people think about something more like this? (screenshots are from my local) 

<img width="1570" alt="Screen Shot 2023-01-26 at 1 32 36 PM" src="https://user-images.githubusercontent.com/397895/214955343-2c723a4e-997e-4afd-b74b-020b4ae72330.png">
<img width="1570" alt="Screen Shot 2023-01-26 at 1 33 11 PM" src="https://user-images.githubusercontent.com/397895/214955439-bcd37077-0018-4fe9-83a0-f8fa52b2c091.png">
<img width="1569" alt="Screen Shot 2023-01-26 at 1 33 32 PM" src="https://user-images.githubusercontent.com/397895/214955496-5565b080-112d-431f-85e7-eae08fb04b60.png">
<img width="1573" alt="Screen Shot 2023-01-26 at 1 37 33 PM" src="https://user-images.githubusercontent.com/397895/214956224-079f3129-2ead-4b46-b781-9562359f5345.png">
